src/GitReportCL/RunProcess.cs

```diff
@@ -0,0 +1,105 @@
+            int count = 0;
+            System.Text.Encoding encoding = System.Text.Encoding.UTF8;
+            encoding.GetBytes(ProcessDiff());
+            while ( byteArray[count] = encoding.GetBytes(process.StandardOutput.ReadLine()))
+            string processTemp = process.StandardOutput.ReadToEnd();
+            //Console.WriteLine("Co proces wyrzuca: {0}",processTemp);
+            process.CloseMainWindow();
+            process.Dispose();
+            return processTemp;
+        }*/
+        private string ArgumentsForBefore()
+        {
+            //Console.WriteLine("EndDate: {0}", validate.setGitRep.gitArguments[1].Substring(0,8));
+            //Console.WriteLine("log --pretty=\"%H\" --before=\"24:00 " + validate.setGitRep.gitArguments[1].Substring(0, 8) + "\" -1");
+            return "log --pretty=\"%H\" --before=\"24:00 " + validate.setGitRep.gitArguments[1].Substring(0, 8) + "\" -1";
+        }
+        private string ArgumentsForSince()
+        {
+            //Console.WriteLine("StartDate: {0}", validate.setGitRep.gitArguments[0].Substring(0, 8));
+            // Console.WriteLine("log --pretty=\"%H\" --before=\"00:00 " + validate.setGitRep.gitArguments[0].Substring(0, 8) + "\" -1");
+            return "log --pretty=\"%H\" --before=\"00:00 " + validate.setGitRep.gitArguments[0].Substring(0, 8) + "\" -1";
+        }
+        private string BuildArg(string commitSince, string commitBefore)
+        {
+            //Console.WriteLine("diff --numstat " + commitSince.Substring(0, 10) + ".." + commitBefore.Substring(0, 10));
+            return "diff --numstat " + commitSince.Substring(0, 10) + ".." + commitBefore.Substring(0, 10);
+        }
+        public string ProcessDiff()
+        {
+            //Console.WriteLine("ComitSince 1:{0} i Comitbefore 2:{1}",ProcessInitialization(ArgumentsForSince()), ProcessInitialization(ArgumentsForBefore()));
+            return ProcessInitialization(BuildArg(ProcessInitialization(ArgumentsForSince()), ProcessInitialization(ArgumentsForBefore())));
+        }
+        public char[] ConvertToChar(string arg)
+        {
+            return arg.ToCharArray();
+        }
+        public void Process()
+        {
+            System.Text.Encoding encoding = System.Text.Encoding.UTF8;
+
+            if (encoding.GetMaxByteCount(ConvertToChar(ProcessDiff()).Length) < int.MaxValue)
+            {
+                {
+                    byte[] byteArray = encoding.GetBytes(ProcessDiff());
+
+                    using (MemoryStream memStream = new MemoryStream(byteArray))
+                    {
+                        using (FileStream fileStream = new FileStream("C:\\Testy_C#\\test.1.txt", FileMode.Create))
+                        {
+                            memStream.CopyTo(fileStream);
+                        }
+                    }
+                    string myString = new string(encoding.GetChars(byteArray));               //przepisanie bajtow do stringa
```

Comment:
* `myString` is not descriptive, change to something more descriptive.
* comments should be in english language.